test/jdk/java/foreign/TestLinker.java

```diff
@@ -24,7 +24,6 @@
 /*
  * @test
  * @enablePreview
- * @requires jdk.foreign.linker != "UNSUPPORTED"
```

Comment:
This shouldn't be removed. (Note that the fallback linker is `FALLBACK`, so this test was already enabled for it).

Reply:
Apologies - I was fighting with jtreg to get the fallback linker enabled (so I could test). This line got in between :-)